include/mbedtls/ssl.h

```diff
@@ -2031,6 +2038,19 @@ void mbedtls_ssl_conf_extended_master_secret( mbedtls_ssl_config *conf, char ems
 void mbedtls_ssl_conf_arc4_support( mbedtls_ssl_config *conf, char arc4 );
 #endif /* MBEDTLS_ARC4_C */
 
+#if defined(MBEDTLS_SSL_SRV_C)
+/**
+ * \brief          Disable or enable CA list suppression
```

Comment:
Thanks for changing the name of the constants, please update the documentation to match (e.g. here “Whether to send a list of acceptable CAs in CertificateRequest messages (default: do send)”).